docs/concepts/services-networking/network-policies.md

```diff
@@ -62,6 +77,13 @@ __podSelector__: Each `NetworkPolicy` includes a `podSelector` which selects the
 
 __ingress__: Each `NetworkPolicy` includes a list of whitelist `ingress` rules.  Each rule allows traffic which matches both the `from` and `ports` sections. The example policy contains a single rule, which matches traffic on a single port, from either of two sources, the first specified via a `namespaceSelector` and the second specified via a `podSelector`.
 
+__egress__: Each `NetworkPolicy` includes a list of whitelist `egress` rules. Each rule allows traffic which matches both the `ports` and `to` sections. The example policy contains a single rule, which matches traffic on a single port, to either of two sources, the first specified via a `namespaceSelector` and the second specified via a `podSelector`.
+
+__ipBlock__: `ipBlock` describes a particular CIDR that is allowed to
```

Comment:
This is a bit asymmetrical - we don't document the `podSelector` and `namespaceSelector` bits here, which are equivalent. 

Not sure if we care though?